metabase/metabase
# [Cards in Dashboards] Update in-dashboard question creation flow

Closes #51169

### Description

Implements updated design that moves the ability to create new questions on a dashboard into the question picker sidebar. Also fixes a mistake where we didn't take into account the user's permissions when showing the new buttons (thanks for catching @kdoh!).

### How to verify

1. Go to a dashboard and edit
2. Click the `+` button to add a new card
3. You will see buttons for "New question" and "New SQL query" (try each flow)
4. Sign in as a user who has no write permissions, see what only "New question" appears
5. 

### Demo

User with access demo (video)
https://github.com/user-attachments/assets/7852426a-2956-46ce-a5b8-68140db2f4ed

User with query builder only access sees:
![CleanShot 2024-12-16 at 18 40 53@2x](https://github.com/user-attachments/assets/c4d045ae-6316-4fcb-a2bf-67fd78776adc)

User with no query permission access sees:
![CleanShot 2024-12-16 at 18 37 28@2x](https://github.com/user-attachments/assets/9257db61-927f-4cba-b2ef-e06d61bad9ef)

### Checklist
